po/ru.po

```diff
@@ -76,9 +76,7 @@ msgstr "Сбрость"
 
 #: src/resources/ui/preferences.ui:84
 msgid "The cat tells you the CPU usage by running speed"
-msgstr ""
```

Comment:
I am not an expert on GNU gettext files but the reason why I opened this PR is because the string for the translation was outside the strings of msgstr. I might be wrong though.